devices/pine64-pinephone/kernel/config.aarch64

```diff
@@ -2719,7 +2791,7 @@ CONFIG_INPUT_GPIO_VIBRA=y
 # CONFIG_INPUT_CM109 is not set
 # CONFIG_INPUT_REGULATOR_HAPTIC is not set
 CONFIG_INPUT_AXP20X_PEK=y
-# CONFIG_INPUT_UINPUT is not set
+CONFIG_INPUT_UINPUT=y
```

Comment:
I manually added this, per https://github.com/NixOS/mobile-nixos/pull/449#issuecomment-1030914656.